.github/CODEOWNERS

```diff
@@ -1,6 +1,101 @@
 # Per-directory ownership and automatic assignment for pull requests.
+
+# Audio and Translation team.
+/core/templates/dev/head/exploration_editor/translation_tab/ @DubeySandeep
+/core/controllers/translator*.py @DubeySandeep
+
+
+# Collection project.
+/core/domain/collection*.py @kevinlee12
+/core/template/dev/head/collection_editor/ @kevinlee12
+/core/template/dev/head/collection_player/ @kevinlee12
+
+
+# Dev workflow team.
+/core/tests/ @apb7
+/scripts/ @apb7
+
+
+# Lesson Analytics team.
+/core/domain/stats*.py @brianrodri
+/core/storage/statistics/ @brianrodri
+/core/templates/exploration_editor/stats_tab/ @brianrodri
+/extensions/actions/ @brianrodri
+/extensions/visualizations/ @brianrodri
+/extensions/issues/ @brianrodri
+
+
+# Library page.
+/core/template/dev/head/library/ @kevinlee12
+/core/domain/library_services.py @kevinlee12
+
+
+# New structures team.
+# Question project.
+/core/controllers/question*.py @aks681
+/core/domain/question*.py  @aks681
+/core/storage/question/ @aks681
+/core/templates/dev/head/domain/question/ @aks681
+/core/templates/dev/head/pages/question_editor/ @aks681
+/core/templates/dev/head/pages/questions_list/ @aks681
+
+# Skill project.
+/core/controllers/skill*.py @aks681
+/core/domain/skill*.py  @aks681
+/core/storage/skill/ @aks681
+/core/templates/dev/head/domain/skill/ @aks681
+/core/templates/dev/head/pages/skill*/ @aks681
+
+# Story project.
+/core/controllers/story*.py @aks681
+/core/domain/story*.py  @aks681
+/core/storage/story/ @aks681
+/core/templates/dev/head/domain/story/ @aks681
+/core/templates/dev/head/pages/story*/ @aks681
+
+# Topic project.
+/core/controllers/topic_*.py @aks681
+/core/domain/topic*.py @aks681
+/core/storage/topic/ @aks681
+/core/templates/dev/head/domain/topic/ @aks681
+/core/templates/dev/head/pages/topic_*/ @aks681
```

Comment:
@seanlip, I didn't change this `_*` to separate it from `topics_and_skills_dashboard`.

Reply:
SGTM, but could you add an explanation as a comment? Future devs won't know, otherwise.

Reply:
Done! (Added a comment explaining why it's required.)